### ISSUE TITLE
REFL018 should warn here

### ISSUE THREAD

**Issue by JohanLarsson** [DotNetAnalyzers/ReflectionAnalyzers#123]
```cs
typeof(string).GetMethod("ToBoolean")
```
